src/libp2p_stream_mplex_worker.erl

```diff
@@ -88,9 +89,15 @@ init(
             kind = Kind
         }
     end,
+    lager:debug("starting mplex worker for streamid ~p, mod ~p, stack ~p", [StreamID, Mod, CompleteStack]),
```

Comment:
Is this not basically the same as the above notice? Maybe just have one of them at the debug level?